libs/langgraph/src/pregel/algo.ts

```diff
@@ -64,7 +75,7 @@ export type WritesProtocol<C = string> = {
   name: string;
   writes: PendingWrite<C>[];
   triggers: string[];
-  path?: [string, ...(string | number)[]];
+  path?: TaskPath;
```

Comment:
Should have refactored this ages ago, thanks

Reply:
np - there's always _something_ that needs polishing!